docs/tutorials/python/evaluation.md

```diff
@@ -0,0 +1,55 @@
+# Evaluations
```

Comment:
I think It would help to include an example showing how to consume the generator returned by one of these methods: get_all_evaluations, get_available_evaluations, or get_evaluations_by_project.